### PR TITLE
Switch to Alpine for Base

### PR DESCRIPTION
**Pull request by kylemanna**
Stumbled on Alpine the other day and it significantly reduces the size of the Docker image. (150 -> 12MB).

I'm running it on three of my existing personal VPNs without issue.  To use it with systemd it's as simple as installing the [systemd service file](https://github.com/kylemanna/docker-openvpn/blob/master/init/docker-openvpn%40.service) and then overriding the image with something like.

    # /etc/systemd/system/docker-openvpn@vpn0.service.d/local.conf
    Environment="IMG=kylemanna/openvpn:alpine"

Run `systemd daemon-reload` and then restart the service.

Random [babbling on my blog](http://bit.ly/1XHb5nB)

Thanks to @tsing I was able to pull in the EasyRSA package which should include package signature verification at @ypid's request.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,21 +1,16 @@
...
 
-RUN apt-get update && \
-    apt-get install -y openvpn iptables curl && \
-    apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
-
-RUN mkdir -p /usr/local/share/easy-rsa && \
-    curl -L https://github.com/OpenVPN/easy-rsa/archive/v3.0.0.tar.gz | tar xzf - --strip=1 -C /usr/local/share/easy-rsa easy-rsa-3.0.0/easyrsa3 && \
-    ln -s /usr/local/share/easy-rsa/easyrsa3/easyrsa /usr/local/bin
+RUN echo "http://dl-4.alpinelinux.org/alpine/edge/testing/" >> /etc/apk/repositories && \
+    apk add --update openvpn iptables curl bash easy-rsa && \
```
Comment (tsing) [inline Dockerfile]: It seems that the `curl` is not really used ?

Comment (kylemanna) [inline Dockerfile]: Correct. After your testing/EasyRSA it's unneeded.  Let me fix this.

Comment (dx9) [inline Dockerfile]: path needs fixing

docker run --volumes-from ovpn-data --rm -it kylemanna/openvpn:alpine ovpn_initpki                    
/usr/local/bin/ovpn_initpki: line 19: easyrsa: command not found

Comment (kylemanna) [inline Dockerfile]: My fault again for not testing. :flushed:  Added a symlink and updated the EasyRSA environment.

Thanks for testing!

**Comment by kylemanna** [comment]
This has been sitting quiet for a while.  Just tagged v1.1.0 on current master (Docker Hub tag: 1.1.0).

Thinking about merging in the next few days.  The Travis-CI tests tell me it works. :+1: 

If things went sideways people could use the 1.1.0 tag while master resolves any last minute surprises.

Thoughts?

**Pull request merged**